python/ql/src/semmle/python/ApiGraphs.qll

```diff
@@ -351,20 +351,95 @@ module API {
 
     private import semmle.python.types.Builtins as Builtins
```

Comment:
```suggestion
```

As I see it, we're not using this import anymore, and I don't think there are any side-effects from having the classes from this module in scope that we care about?

Reply:
Good catch. Indeed, there is no reason to keep this around.